type.py

```diff
@@ -86,21 +86,19 @@ def get_configs(self):
         return vars(self)
 
 
-""" Run Configs """
+""" Experiment """
 
 
 @dataclass
-class RunConfig:
+class Experiment:
+    project_name: str
     run_name: str  # Get's appended as a prefix before the pipeline name
     train: bool  # Weather the run should do training
     dataset: pd.DataFrame
+    pipeline: "Pipeline"
+    metrics: Evaluators
+    preprocessing_config: PreprocessConfig
```

Comment:
so actually there could be multiple preprocessing_configs, if we merge the data. so this is in theory incorrect, in practice, it's probably fine to pass in the one we want to log.

Reply:
You mean in case there are multiple initial data sources?

Reply:
right now, multiple ones that we merge together.
But later, there'll be multiple datasources, I completely forgot about that as well!